### ISSUE TITLE
Unknown info subpacket type 05: JA-111A

### ISSUE THREAD

**Issue by andyboeh** [kukulich/home-assistant-jablotron100#57]
Hi,

we have a JA-111A outdoor siren which doesn't seem to be integrated correctly:

HA gives the following log message:
```
Source: custom_components/jablotron100/jablotron.py:1015

Unknown info subpacket type 05 (device 60): 90053c050200c6
Unknown info subpacket type 05 (device 60): 90193c050200c6050200c6050200c6050200c6050200c6050200c6
```

The sensor `Problem` always reports "Problem" while there is no actual problem with the device (according to F-Link).

Is the JA-111A supported at all?

Thanks

**Comment by andyboeh** [comment]
I just noticed that the sensor "Problem" doesn't have a history longer than the last update, was it introduced just recently?

**Comment by kukulich** [comment]
@andyboeh Please enable debug for devices and try to log some data for the device. We need to log some packets starting with `55`.

**Comment by kukulich** [comment]
The unknown packet should not be logged in “master” version.

**Comment by kukulich** [comment]
Released in 3.15.0.
